### PR TITLE
zunit cleanup

### PR DESCRIPTION
**Pull request by dennis-behm**
- uses the standard boolean representation
    a) single cmdline option, b) application-conf uses `true` rather than `True`
- fixes the classification return codes reported from the zunit runner
@jbyibm @drbruce-git

### CODE REVIEW

File: build.groovy [hunk 1]
```diff
@@ -39,7 +39,7 @@ else {
 		println("** Invoking build scripts according to build order: ${props.buildOrder}")
 		String[] buildOrderList = props.buildOrder.split(',')
 		String[] testOrderList;
-		if (props.runzTests == "True") { 
+		if (props.runzTests) {
```
Comment (jbyibm) [inline build.groovy:42]: It is enough ? Or should we have this ?
if (props.runzTests && props.runzTests .toBoolean())

File: utilities/BuildUtilities.groovy [hunk 2]
```diff
@@ -331,7 +331,7 @@ def relativizeFolderPath(String folder, String path) {
  * getScannerInstantiates - returns the mapped scanner or default scanner
  */
 def getScanner(String buildFile){
-	if (props.runzTests == "True") {
+	if (props.runzTests) {
```
Comment (jbyibm) [inline utilities/BuildUtilities.groovy:334]: It is enough ? Or should we have this ?
if (props.runzTests && props.runzTests .toBoolean())

File: languages/Cobol.groovy [hunk 3]
```diff
@@ -17,10 +17,11 @@ println("** Building files mapped to ${this.class.getName()}.groovy script")
 // verify required build properties
 buildUtils.assertBuildProperties(props.cobol_requiredBuildProperties)
 
+// create language datasets
 def langQualifier = "cobol"
 buildUtils.createLanguageDatasets(langQualifier)
 
-if (props.runzTests == "True") {
+if (props.runzTests) {
```
Comment (jbyibm) [inline languages/Cobol.groovy:24]: It is enough ? Or should we have this ?
if (props.runzTests && props.runzTests .toBoolean())

**Review by jbyibm — CHANGES_REQUESTED** [review]

File: build.groovy (hunk 1 above)
Comment (dennis-behm) [reply build.groovy]: Agreed, we need to test the boolean value of it!

**Comment by dennis-behm** [comment]
@jbyibm requested changes delivered

**Review by jbyibm — APPROVED** [review]

**Comment by jbyibm** [comment]
@dennis-behm It is possible to merge this PR ?

**Pull request merged**